sdk/storage/azure-storage-file-datalake/tests/testcase.py

```diff
@@ -172,73 +146,22 @@ def _set_test_proxy(service, settings):
                 settings.PROXY_PASSWORD,
             )
 
-    def _get_shared_key_credential(self):
+    def _get_shared_key_credential(self, account_name, account_key):
         return {
-            "account_name": self.settings.STORAGE_DATA_LAKE_ACCOUNT_NAME,
-            "account_key": self.settings.STORAGE_DATA_LAKE_ACCOUNT_KEY
+            "account_name": account_name,
```

Comment:
it's a type of credential we can pass in DataLakeServiceClient etc.
It seems currently we are not using this _get_shared_key_credential, an we passed account key as credential.
While it would be better if we can use _get_shared_key_credential so the credential is in dict format, this helps especially for customized url. Since with a pure account key we cannot parse out the account name for user.

Reply:
that makes sense!